src/thunderbots/software/ai/primitive/move_primitive.cpp

```diff
@@ -48,10 +56,24 @@ double MovePrimitive::getFinalSpeed() const
     return final_speed;
 }
 
+bool MovePrimitive::getAutoKickEnabled() const
```

Comment:
I know where this function name is coming from, but I feel like it can be easily mistaken as a function that 'gets auto-kick enabled'. Same with getDribberEnabled()

Reply:
imo getters for Booleans work better with `is` instead of `get` i.e. `isDribblerEnabled()` and `isAutoKickEnalbed()`

Reply:
I totally agree. Changed.